package.json

```diff
@@ -169,6 +169,7 @@
     "calculate-size": "^1.1.1",
     "chroma-js": "^1.3.6",
     "classnames": "^2.2.3",
+    "clipboardy": "^2.3.0",
```

Comment:
If this is only being used for testing we should make it a devDependency

Reply:
Good point.  Changed.